### PR TITLE
shantea controls and opendeck definitions

### PR DESCRIPTION
**Pull request by paradajz**
added shantea controls org definition
added opendeck device

### CODE REVIEW

File: 1209/8472/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: OpenDeck
+owner: Shantea Controls
+license: GPLv3
+site: http://shanteacontrols.wordpress.com
+source: http://github.com/paradajz/OpenDeck
+---
```
Comment (Arachnid) [inline 1209/8472/index.md]: Could you please add a short description of the device here?

File: 1209/8472/index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+layout: pid
+title: OpenDeck
+owner: Shantea Controls
```
Comment (Arachnid) [inline 1209/8472/index.md]: This needs to be the same as the directory name of your org - 'shanteacontrols'.

File: 1209/8472/index.md (hunk 1 above)
Comment (paradajz) [inline 1209/8472/index.md]: done

File: 1209/8472/index.md (hunk 2 above)
Comment (paradajz) [inline 1209/8472/index.md]: fixed!

**Pull request merged**